lib/tasks/achievements_import.rake

```diff
@@ -0,0 +1,15 @@
+namespace :desperado do
+
+  desc 'Import achievements into the DB.'
+  task :achievements_import, [:data_file] => :environment do |_, args|
+    data_file = "#{Rails.root}/#{args.data_file}"
+    File.open(data_file, 'r') do |data|
+      achievements = Psych.load_stream data
+      Achievement.transaction do
+        Achievement.create! achievements
+      end
```

Comment:
> I have a feeling that ActiveRecord is clever enough to create multiple objects if you pass an array of hashes. Worth a shot at least.

[You are right!](http://api.rubyonrails.org/classes/ActiveRecord/Persistence/ClassMethods.html#method-i-create)

Johno, you are a :star: I have updated the code to match your guidance. Thanks! :+1: